cardano-api/src/Cardano/Api/Types.hs

```diff
@@ -35,8 +41,8 @@ data KeyPair
   -- An 'XPub' is 32 bytes of public key and 32 bytes of Chaincode which is used in the
   -- Byron address derivation scheme.
   = KeyPairByron !Crypto.VerificationKey !Crypto.SigningKey
-  | KeyPairShelley -- Exact crypto not ready yet.
-  deriving (Generic, NFData, Show)
+  | KeyPairShelley !(VKey TPraosStandardCrypto) !(SKey TPraosStandardCrypto)
```

Comment:
Should I use the `cardano-ledger-specs` types (`VKey` and `SKey`) or stick with `VerKeyDSIGN (DSIGN TPraosStandardCrypto)` and `SignKeyDSIGN (DSIGN TPraosStandardCrypto)`?